src/common/oclengine.cpp

```diff
@@ -125,81 +235,44 @@ void OCLEngine::InitOCL()
         std::shared_ptr<OCLDeviceContext> devCntxt = std::make_shared<OCLDeviceContext>(
             devPlatVec[i], all_devices[i], all_contexts[all_contexts.size() - 1], plat_id);
 
-        cl::Program program = cl::Program(devCntxt->context, sources);
+        std::string fileName = "qrack_ocl_dev_" + std::to_string(i) + ".ir";
```

Comment:
Define this as a constant at the top of the file, including the extension.

Reply:
Done.